### ISSUE TITLE
orphaned gst-launch processes hang around forever

### ISSUE THREAD

**Issue by karlkleinpaste** [Xpra-org/xpra#3165]
f34 server, 4.3-10.r29154.
what's with the freakish number of stuck gst-launch processes? some from sessions that came and went days before, or that were briefly lived yesterday as i tried the resize-display patch? note start dates, and all using some amount of cpu time.
```
ps aux | egrep xpra\|PID
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
karl      166808  0.0  0.2 808696 43500 ?        SLl  Jun09   2:14 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      170098  0.0  0.2 808728 43444 ?        SLl  Jun09   2:12 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      171022  0.0  0.2 808696 43676 ?        SLl  Jun09   2:12 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      175051  0.0  0.2 808728 43108 ?        SLl  Jun09   2:09 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      194515  0.0  0.2 808808 46216 ?        SLl  Jun10   1:36 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      198327  0.0  0.2 815456 48088 ?        SLl  Jun10   1:32 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      198554  0.0  0.2 815456 47936 ?        SLl  Jun10   1:32 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      199629  0.0  0.2 808412 45396 ?        SLl  Jun10   1:32 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      199858 12.9  2.9 3932512 476160 ?      Ssl  Jun10 324:00 /usr/libexec/Xorg -noreset -novtswitch -nolisten tcp +extension GLX +extension RANDR +extension RENDER -auth /home/karl/.Xauthority -logfile /run/user/5271/xpra/Xorg.:43.log -configdir /run/user/5271/xpra/xorg.conf.d/199857 -config /etc/xpra/xorg.conf -depth 24 :43
karl      200644  0.0  0.2 808412 47264 ?        SLl  Jun10   1:32 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      201363  0.0  0.2 808412 47296 ?        SLl  Jun10   1:32 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      201503  0.0  0.2 808412 47484 ?        SLl  Jun10   1:31 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      208389  0.0  0.2 808412 47600 ?        SLl  Jun10   1:17 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      212290 14.3  1.6 9670980 268012 ?      Sl   Jun11 274:25 /usr/bin/python3.9 /usr/bin/xpra start-desktop :43 --start-child=vglrun mate-session --resize-display=1856x1024 --systemd-run=no
karl      212877  0.2  0.1 415540 26848 ?        Ssl  Jun11   5:24 /usr/bin/python3 /usr/bin/xpra _proxy :43
karl      212911  0.0  0.2 808444 47420 ?        SLl  Jun11   1:09 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      232734  0.0  0.2 808780 47316 ?        SLl  Jun11   0:33 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      233025  0.0  0.2 808780 47756 ?        SLl  Jun11   0:34 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
karl      233286  0.0  0.2 808780 46108 ?        SLl  Jun11   0:34 gst-launch-1.0 -q filesrc location=/usr/share/xpra/bell.wav ! decodebin ! audioconvert ! alsasink
```

**Comment by totaam** [comment]
It's a bug.
One that I had seen before but forgot to make a ticket for.
Until this is fixed, you can workaround the problem using:
```
xpra start --env=XPRA_NEW_STREAM_SOUND=0 ...
```

**Comment by totaam** [comment]
The latest Fedora beta builds should have the fix above. Please close this ticket if that works for you.

(this won't help in cases where the server crashes hard.. not much we can do about that for now)

**Comment by totaam** [comment]
Not heard back so this was not included in 4.2.1, it will be in 4.2.2
